### DISCUSSION TITLE
Progress Bar at Almost 0% But Model is Trained, Missing steps?

### DISCUSSION THREAD

**Discussion by danielsiryakov** [ThilinaRajapakse/simpletransformers#1233] [Q&A]
Hi Everyone, 

I am running into a weird issue where the first progress bar after train() is stopping at a very early stage, always at the same number, but then the epoch itself starts. What does this mean? Am i missing some steps here? I suspect it has something to do with the tokenization of the data but no real documents are present on it. 

![Capture](https://user-images.githubusercontent.com/9343692/129090882-46531390-a8dd-4107-b4d7-4ec364f495f5.PNG)
Thanks!

**Comment by glher** [comment]
This is linked to issue #1302.

I encountered the same problem, and looked at it in a bit more details, and it seems to be a tqdm/Notebook issue (as noted by Thilina). The cached file that is created (e.g. cached_train_roberta_128_2_2) does contain the entire dataset. One can test it by downloading the cached file from Google Colab, and doing:

    import torch
    data = torch.load('cached_train_roberta_128_2_2')
    print(data[0]['input_ids'].size())

**Comment by UmutAlihan** [comment]
so is there any solution attempt? I am also having this issue and it is quite annoying unfortunately
